example/index.js

```diff
@@ -269,11 +278,20 @@ function onParamsChange() {
 
 	}
 
+	// Denoiser
+	else pathTracer.maxSamples = params.maxSamples;
```

Comment:
Why are we conditionally setting max samples here?

Reply:
leftover, will remove.